soccer/Robot.cpp

```diff
@@ -640,11 +641,50 @@ void OurRobot::replanIfNeeded(const Geometry2d::CompositeShape& global_obstacles
 	if (!_pathInvalidated) {
 		addText("Reusing path");
 	} else {
-		Planning::Path *newlyPlannedPath = _planner->run(pos, angle, vel, _motionConstraints, &full_obstacles);
+		/*
+		MotionConstraints test = _motionConstraints;
+		test.targetPos = ((*test.targetPos - pos)/2.0) + pos;
+		if (!test.targetWorldVel) {
+			test.targetWorldVel = Point(0,0);
+		}
+		*/
+		
+		Planning::Path *first = _planner->run(pos, angle, vel, _motionConstraints, &full_obstacles);
+		//unique_ptr<Planning::Path> subPath = first->subPath(0,first->getTime()/2.0);
+		//unique_ptr<Planning::Path> subPath2 = first->subPath(first->getTime()/2.0,first->getTime());
+		
+		//Planning::Path *first = _planner->run(pos, angle, vel, test, &full_obstacles);
+		//Planning::Path *second = _planner->run(*test.targetPos, angle, *test.targetWorldVel, _motionConstraints, &full_obstacles);
+		//Planning::CompositePath *composite=new Planning::CompositePath();
+		
+		/*
```

Comment:
Go ahead and remove all the commented-out old code - it's fine for development, but shouldn't really get merged into master.